web/satellite/src/api/projectMembers.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import apollo from '@/utils/apolloManager';
+import gql from "graphql-tag";
+
+// Performs graqhQL request.
+// Throws an exception if error occurs
+export async function addProjectMember(userID: string, projectID: string): Promise<any> {
+    let response = await apollo.mutate(
+        {
+            mutation: gql(`
+                mutation {
+                    addProjectMember(
+                        projectID: "${projectID}",
+                        userID: "${userID}"
+                    ) {id}
+                }`
+            ),
+            fetchPolicy: "no-cache",
+        }
+    );
+
+    if(!response){
+        // TODO: replace with popup in future
+        console.log("cannot create project");
```

Comment:
At the beginning of the next iteration, we will start using linter which forbids using `console.log` use `console.error` instead